go.mod

```diff
@@ -8,32 +8,30 @@ require (
 	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
 	github.com/iancoleman/strcase v0.2.0
 	github.com/kr/text v0.2.0
-	github.com/onsi/ginkgo/v2 v2.1.6
-	github.com/onsi/gomega v1.20.1
-	github.com/operator-framework/operator-lib v0.11.0
-	github.com/prometheus/client_golang v1.13.0
+	github.com/onsi/ginkgo/v2 v2.7.0
+	github.com/onsi/gomega v1.24.2
+	github.com/operator-framework/operator-lib v0.11.1-0.20230126194151-7fc7204a9445
```

Comment:
Need to merge operator-lib [PR](https://github.com/operator-framework/operator-lib/pull/112) first and then bump `operator-lib` here to point it to the latest commit that references updated k8s dependencies.